### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by williamboman-pp**

### CODE REVIEW

File: product-library.adoc [hunk 1]
```diff
@@ -21,40 +21,83 @@ The Product Library API implements the following scopes:
...
-      }
-    }
-  ],
-  "externalReference": "externalReference",
-  "unitName": "Kg",
-  "vatPercentage": "10"
+	"uuid": "89f9d200-fb8a-11ea-adc1-0242ac120002",
+	"name": "Jeans",
+	"description": "Description of product.",
+    "imageLookupKeys": [
```
Comment (williamboman-pp) [inline product-library.adoc:32]: @pypl-simon I can't remember if imageLookupKeys or presentation.imageUrl is preferred?

**Review by martinfridh — APPROVED** [review]
LGTM! 

Nice addition with public changelog also. Is this changelog be referenced from anywhere so integrators can find it?

File: product-library.adoc (hunk 1 above)
Comment (pypl-simon) [reply product-library.adoc:32]: `presentation.imageUrl` is the preferred one. `imageLookupKeys` is deprecated.

File: image.adoc [hunk 2]
```diff
@@ -40,5 +40,6 @@ https://image.izettle.com/swagger
 ```
 Where `o` is a `2000*2000px` and `L` is a `560*560px` image.
 
+To apply this uploaded image on a product, use the first URL provided in the `imageUrls` array.
```
Comment (williamboman-pp) [inline image.adoc:43]: @pp-svanstrom Would you say this is correct?

**Comment by williamboman-pp** [comment]
> Nice addition with public changelog also. Is this changelog be referenced from anywhere so integrators can find it?

Maybe at some point, but not immediately no. It's just adding a bit more structure to the change management process, we'll see how it can be adopted for future workflows.

File: image.adoc (hunk 2 above)
Comment (pp-svanstrom) [reply image.adoc:43]: Yes 👍

**Review by pp-svanstrom — APPROVED** [review]
Nice 👍  Creating a change-log is a brilliant idea!

**Pull request merged**